theme/partials/social.html

```diff
@@ -0,0 +1,9 @@
+{% if config.extra.social %}
+  <div class="md-footer-social">
+    <link rel="stylesheet" href="{{ base_url }}/assets/fonts/font-awesome.css">
```

Comment:
This is a pretty heavy dependency for only a couple if small icons IMO

Reply:
Agree. That's however from mkdocs-material, but we can ofc change it